package.json

```diff
@@ -51,6 +51,7 @@
     "@web-stories-wp/stories-block": "*",
     "@web-stories-wp/stories-carousel": "*",
     "@web-stories-wp/stories-lightbox": "*",
+    "@web-stories-wp/react": "*",
```

Comment:
Can remove the `use-debounce` dependency from this file.

Can also remove `use-reduction` as this doesn't seem to be used anywhere, actually.